src/js/auth/containers/AuthApp.jsx

```diff
@@ -84,7 +84,7 @@ class AuthApp extends React.Component {
         <div>
           <h3>We are sorry that we could not successfully log you in.</h3>
           <h3>Please call the Vets.gov Help Desk at 1-855-574-7286. We're open Monday‒Friday, 8:00 a.m.‒8:00 p.m. (ET).</h3>
-          <button onClick={window.close()}>Close</button>
+          <button onClick={window.close}>Close</button>
```

Comment:
Sorry for the delay - I did some quick manual testing to see if I could get `window.close` to call on the wrong `window` and couldn't do it.  I think both `window.close` & `window.close.bind(window)` are equivalent here.  I'll yield to whatever you think is better.